src/java/org/apache/cassandra/tools/nodetool/Sjk.java

```diff
@@ -143,7 +145,7 @@ else if (isListCommands())
                 {
                     for (String cmd : commands.keySet())
                     {
-                        System.out.println(String.format("%8s - %s", cmd, parser.getCommandDescription(cmd)));
```

Comment:
missing `parser.usage`, you can override it but sadly its annoying.

`parser.usage();` replace with

```
StringBuilder sb = new StringBuilder();
parser.usage(sb);
outStream.println(sb);
```

and `parser.usage(cmd);` replace with 

```
StringBuilder sb = new StringBuilder();
parser.usage(cmd, sb);
outStream.println(sb);
```